client/html/templates/email/common/text-summary-detail-body-default.php

```diff
@@ -73,6 +73,7 @@
 <?php	$price = $product->getPrice(); ?>
 
 <?php echo strip_tags( $product->getName() ); ?>
+<?php echo strip_tags( $product->getProductCode() ); ?>
```

Comment:
`strip_tags()` may be problematic in this case if the codes contain a "<" or ">" character (for whatever reason). Can you remove the function?